src/account-multicall/LlamaAccountMulticallExtension.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+import {LlamaAccountMulticallStorage} from "src/account-multicall/LlamaAccountMulticallStorage.sol";
+import {LlamaBaseAccountExtension} from "src/common/LlamaBaseAccountExtension.sol";
+import {LlamaUtils} from "src/lib/LlamaUtils.sol";
+
+/// @title Llama Account Multicall Extension
+/// @author Llama (devsdosomething@llama.xyz)
+/// @notice An account extension that can multicall on behalf of the Llama account.
+/// @dev This contract should be delegate-called from a Llama account.
```

Comment:
```suggestion
/// @dev This contract should be delegatecalled from a Llama account.
```

Reply:
Done in [681bf1f](https://github.com/llamaxyz/llama-periphery/pull/100/commits/681bf1f43330c1098e1d2c25e90fbc907f2d140e)